includes/admin/class-wcmp-export.php

```diff
@@ -1626,6 +1633,40 @@ function (
                 }
             );
     }
+
+    /**
+     * @param string $returnOptions
+     *
+     * @throws ApiException
+     * @throws MissingFieldException
+     */
+    public function addReturnInTheBox(string $returnOptions)
+    {
+        $this->myParcelCollection
+            ->generateReturnConsignments(
+                false,
+                function(
+                    AbstractConsignment $returnConsignment,
+                    AbstractConsignment $parent
+                ) use ($returnOptions): AbstractConsignment {
+                    $returnConsignment->setLabelDescription(
+                        'Return: ' . $parent->getLabelDescription() .
+                        ' This label is valid until: ' . date("d-m-Y", strtotime("+ 28 days"))
+                    );
+
+                    if ($returnOptions === WCMP_Settings_Data::NO_OPTIONS) {
```

Comment:
Boven deed je yoda, hier weer niet. Weet niet of je dit weet maar yoda houdt in dat je de constante links van de operator zet, maakt het iets leesbaarder.